### DISCUSSION TITLE
Does a JS wrapper exist ?

### DISCUSSION THREAD

**Discussion by MonsieurBibo** [FlareSolverr/FlareSolverr#351] [General]
Hello, 

as my title suggests, I would like to know if a JavaScript/TypeScript wrapper exists. If not, is it in your plans to create one ? 
It doesn't seem too difficult because it should only do requests handling. I might even try it myself.

Thanks in advance ! 😀

**Comment by ngosang** [comment]
It does not exist and it's not planned. The project requires an installed web browser and it can be tricky to set up.
You can take the source code and tailor to suit your needs.
